specification/keyvault/Security.KeyVault.EKM/tspconfig.yaml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  "service-dir":
+    default: "sdk/keyvault"
+  "dependencies":
+    default: ""
+
+emit:
+  - "@azure-tools/typespec-autorest"
+
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    output-file: "{azure-resource-provider-folder}/Microsoft.KeyVault/{version-status}/{version}/ekm.json"
+  # Uncomment this line and add "@azure-tools/typespec-python" to your package.json to generate Python code
+  "@azure-tools/typespec-python":
+    flavor: azure
+    "basic-setup-py": true
+    "package-name": "azure-keyvault-ekm"
```

Comment:
In EKM doc, it mentions
> MHSM SDK and AZ CLI should be updated to support following actions

Wondering which one is the MHSM SDK?

Ask because here you seem just write a new package name for Python SDK.